project/src/main/webapp/style.css

```diff
@@ -253,7 +253,28 @@ h3 {
     padding: 2vw;
     transform: rotate(45deg);
   }
+<<<<<<< HEAD
```

Comment:
You might want to delete these leftover merge fragments

Reply:
good catch!